### PR TITLE
Use cargo styles for help text

### PR DESCRIPTION
**Pull request by joshka**
image

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -43,6 +43,7 @@ git2 = { version = "0.19.0", default-features = false }
 indoc = "2.0.5"
 termcolor = "1.4.1"
 unicase = "2.8.0"
+clap-cargo = "0.14.1"
```
Comment (orium) [inline Cargo.toml:46]: Needs to be put in alphabetical order.

Comment (joshka) [reply Cargo.toml]: Reordered all the crates in 47b30c5

**Comment by orium** [comment]
You need to rebase on `main`. Can you also squash your commits into a single one please?

**Comment by joshka** [comment]
> You need to rebase on `main`. Can you also squash your commits into a single one please?

Done

**Comment by orium** [comment]
Thanks for your contribution.

**Pull request merged**